src/chain/tezos/michelsonParsingUtil.ts

```diff
@@ -0,0 +1,11 @@
+const nearley = require("nearley");
+const grammar = require("../../utils/grammars/lexedMichelson.js");
+const util = require('util');
+
+const parser = new nearley.Parser(nearley.Grammar.fromCompiled(grammar));
+
+export function michelsonToJson(michelson: string): string {
```

Comment:
Please add new functions of the following form:

```typescript

function michelsonScriptToJson(script: string): Object {
//Implementation
michelsonScriptToJson(parameter, storage, code);
}

function michelsonScriptToJson(parameter: string, storage: string, code: string): Object {
//Implementation
}
```

Here are sample invocations of the code:

```typescript
const json = michelsonScriptToJson("parameter string; storage string; code {CAR; NIL operation; PAIR;};");
const json2 = michelsonScriptToJson("string", "string", "{CAR; NIL operation; PAIR;}";
```

The output should be:
```json
[ { "prim": "parameter", "args": [ { "prim": "string" } ] },
        { "prim": "storage", "args": [ { "prim": "string" } ] },
        { "prim": "code",
          "args":
            [ [ { "prim": "CAR" },
                { "prim": "NIL",
                  "args": [ { "prim": "operation" } ] },
                { "prim": "PAIR" } ] ] } ]
```